### PR TITLE
Add blog post comparing Event Sourcing and CDC.

### PR DESCRIPTION
**Pull request by murphye**
I added a top level URL reference to "/images" for the blog images, so they should resolve correctly from the blog page.

I ran locally and it all rendered OK.

### CODE REVIEW

File: blog/2020-02-06-event-sourcing-vs-cdc.adoc [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+
+The Outbox is also meant to be abstracted from the application as it’s only an ephemeral store of outgoing event data, and not meant to be read or queried. In fact, the Domain Events residing in the Outbox may be deleted immediately after insertion!
+
+== Event Sourcing Journal vs. Outbox
+
+We can now take a closer look at the overlap in design of an Event Sourcing journal and CDC with Outbox. By comparing the attributes of the journal with the Outbox table, the similarities become clear. The Aggregate, again from DDD, is at the heart of how the data is stored and consumed for both Outbox and Event Sourcing.
+
+Here are the common attributes that exist between an Event Sourcing journal and an Outbox:
+
+* Event ID — Unique identifier for the event itself and can be used for deplication for idempotent consumers
```
Comment (gunnarmorling) [inline blog/2020-02-06-event-sourcing-vs-cdc.adoc]:
```suggestion
* Event ID — Unique identifier for the event itself and can be used for de-duplication for idempotent consumers
```

File: blog/2020-02-06-event-sourcing-vs-cdc.adoc [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+
+=== Pros for Event Sourcing with CQRS 
+
+1. Journal is easily accessible for auditing purposes
+2. Generally performant for a high volume of write operations to the Journal
+3. Possibility to shard the Journal for a very large amount of data (depending on datastore)
+
+=== Cons for Event Sourcing with CQRS
+
+1. Everything is eventually consistent data; strongly consistent data doesn’t fit Event Sourcing and CQRS
```
Comment (gunnarmorling) [inline blog/2020-02-06-event-sourcing-vs-cdc.adoc]:
```suggestion
1. Everything is eventually consistent data; a requirement of strongly consistent data doesn’t fit Event Sourcing and CQRS
```

**Comment by murphye** [comment]
I accepted suggestions. Thanks!

**Comment by gunnarmorling** [comment]
Has been merged. Thanks, @murphye!

**Pull request closed without merging**